src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -238,6 +238,10 @@ public static bool TryParseKey(ImmutableArray<byte> blob, out ImmutableArray<byt
                     return true;
                 }
             }
+            catch (OutOfMemoryException)
+            {
```

Comment:
Should we be OOMing in this case? can you comment on why we should instead of failing to parse a key that's too big?

Reply:
it was just an application of the guideline "whenever you catch all exceptions, let OOM go through"